src/main/java/com/onlinebookshop/controller/BookController.java

```diff
@@ -0,0 +1,35 @@
+package com.onlinebookshop.controller;
+
+import com.onlinebookshop.dto.BookDto;
+import com.onlinebookshop.dto.CreateBookRequestDto;
+import com.onlinebookshop.service.BookService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value = "/books")
```

Comment:
By condition it should be:
Book Endpoints: /api/books

Reply:
Agree with Oleg